src/stylesheets/objects/_objects.flex.scss

```diff
@@ -1 +1,4 @@
-
+.o-flex--center {
```

Comment:
`o-flex--center` may not be the best name here to represent flex containers that **center alignment** only. Perhaps something like `o-flex--hcenter` would be better.

Then again, couldn't we just do `u-text-center` to achieve the same thing here?

Reply:
👍